recipes/epics-striptool/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+export TOP="${EPICS_BASE}/extensions"
+export MOTIF_INC="${PREFIX}/include"
+export MOTIF_LIB="${PREFIX}/lib"
+export X11_INC="${PREFIX}/include"
+export X11_LIB="${PREFIX}/lib"
+
+_make="make -j1 V=1 VERBOSE=1 EPICS_BASE=${EPICS_BASE} TOP=${TOP}"
+
+# build
+${_make} build
+
```

Comment:
Is there a make check/test we can run here?

Reply:
@ocefpaf, unfortunately not: https://github.com/epics-extensions/StripTool/blob/StripTool-2-5-17-0/Makefile.